[CI] LuceneTopNSourceOperatorTests testAccumulateSearchLoad failing

**Build Scans:**
- [elasticsearch-intake #35544 / part3](https://gradle-enterprise.elastic.co/s/76kftmzw2bc4m)
- [elasticsearch-intake #35539 / part3](https://gradle-enterprise.elastic.co/s/eoroglqbj4sw2)
- [elasticsearch-pull-request #125483 / part-3](https://gradle-enterprise.elastic.co/s/ftbtougvn63xa)
- [elasticsearch-pull-request #124821 / part-3](https://gradle-enterprise.elastic.co/s/qed2vtmtlocdy)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:compute:test" --tests "org.elasticsearch.compute.lucene.query.LuceneTopNSourceOperatorTests.testAccumulateSearchLoad" -Dtests.seed=EDA92D485428713D -Dtests.locale=ru-KG -Dtests.timezone=Indian/Antananarivo -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.compute.lucene.query.LuceneTopNSourceOperatorTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testAccumulateSearchLoad'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: a value equal to or greater than <0.20143589592617955>
     but: <0.17239370785883648> was less than <0.20143589592617955>
```

**Issue Reasons:**
- [main] 4 failures in test testAccumulateSearchLoad (0.4% fail rate in 934 executions)
- [main] 2 failures in step part-3 (0.3% fail rate in 633 executions)
- [main] 2 failures in step part3 (1.2% fail rate in 164 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (0.3% fail rate in 617 executions)
- [main] 2 failures in pipeline elasticsearch-intake (1.2% fail rate in 164 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -157,15 +157,9 @@ tests:
 - class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeMetricsIT
   method: test
   issue: https://github.com/elastic/elasticsearch/issues/143697
-- class: org.elasticsearch.compute.lucene.query.LuceneTopNSourceOperatorTests
-  method: testAccumulateSearchLoad
-  issue: https://github.com/elastic/elasticsearch/issues/143708
 - class: org.elasticsearch.xpack.sql.qa.security.CliApiKeyIT
   method: testCliConnectionWithInvalidApiKey
   issue: https://github.com/elastic/elasticsearch/issues/143646
-- class: org.elasticsearch.compute.lucene.query.LuceneTopNSourceOperatorScoringTests
-  method: testAccumulateSearchLoad
-  issue: https://github.com/elastic/elasticsearch/issues/143750
 - class: org.elasticsearch.upgrades.FullClusterRestartIT
   method: testClusterState {cluster=UPGRADED}
   issue: https://github.com/elastic/elasticsearch/issues/143800

